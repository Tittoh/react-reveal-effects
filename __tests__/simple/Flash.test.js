/*
 * Flash Component Test Suite
 *
 * Copyright Â© Roman Nosov 2017
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */
import Flash from '../../Flash';

import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Flash', () => {
  it('renders a initial view', () => {
    const content = shallow(
      <Flash>
        <div>Test test</div>
      </Flash>
    );
    expect(content.html()).toMatchSnapshot();
  });
});
