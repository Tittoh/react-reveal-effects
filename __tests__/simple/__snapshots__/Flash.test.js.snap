// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flash renders a initial view 1`] = `"<div>Test test</div>"`;
